.github/workflows/ci-workflow.yml

```diff
@@ -130,7 +130,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [18.x, 20.x, 22.x]
+        node-version: [18.x, 20.x, 22.11]
```

Comment:
I'm trying to create a reproduction case for this so that I can report it to core. I am confident it is due to the unflagging of require ESM.